Branch-SDK/Branch-SDK/BNCCommerceEvent.h

```diff
@@ -11,16 +11,45 @@
 #import "BNCServerRequest.h"
 
 
+#pragma mark BNCProductCategory
+
+typedef NSString*const BNCProductCategory;
```

Comment:
It doesn't matter to the compiler.  Stylistically, there isn't an overwhelming standard. You'll see any of:

```objc
typedef NSString*const BNCProductCategory;
typedef NSString *const BNCProductCategory;
typedef NSString* const BNCProductCategory;
```